recipes/scrapy-contrib-bigexporters/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "scrapy-contrib-bigexporters" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9438c5f6d76bb8d68d38f9472dddab7f4da21abce6da1d320aabdca840046288
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - scrapy >=2.4.0
+  run_constrained:
+    - fastparquet
```

Comment:
What are you trying to accomplish with `run_constrained`? Without version numbers, it's not going to enforce anything. So either need to remove this section or add versions to do the constraining.

Reply:
thank you. I tried to mark them as optional dependencies. I will add the version numbers

Reply:
After reflection, I removed run_constrained